src/components/composite/MultiLayerDrawer/MultiLayerDrawer.stories.tsx

```diff
@@ -0,0 +1,367 @@
+import type { Meta, StoryObj } from '@storybook/nextjs-vite';
+import { MultiLayerDrawer } from './MultiLayerDrawer';
+import {
+  CategoryConfig,
+  CategoryContentType,
+  CategoryOption,
+} from './MultiLayerDrawer.types';
+import { useState } from 'react';
+import Stack from '@mui/material/Stack';
+import { formatSliderValue } from 'src/components/core/form/Select/utils';
+import { usePendingFilters } from './hooks';
+import {
+  chainOptions,
+  protocolOptions,
+  tagOptions,
+  sortOptions,
+  assetOptions,
+} from './fixtures';
+
+const meta: Meta<typeof MultiLayerDrawer> = {
+  title: 'Composite/MultiLayerDrawer',
+  component: MultiLayerDrawer,
+  parameters: {
+    nextjs: {
+      appDirectory: true,
+      navigation: {
+        pathname: '/',
+      },
+    },
+  },
+  tags: ['autodocs'],
+};
+
+export default meta;
+type Story = StoryObj<typeof MultiLayerDrawer>;
+
+// Story 1: Earn Filters (with pending filters pattern)
+const EarnFiltersTemplate = () => {
+  const [appliedFilters, setAppliedFilters] = useState({
```

Comment:
I think that it would be easier to use 

```
interface FilterState {
chains: string[],
...
}
useState<FilterState>()
```

instead of assigning type